components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/AccountLockHandler.java

```diff
@@ -475,9 +479,10 @@ protected boolean handlePostAuthentication(Event event, String userName, UserSto
                     if (unlockTimePropertyValue != 0) {
                         if (log.isDebugEnabled()) {
                             String msg = String.format("Set account unlock time for user: %s in user store: %s " +
-                                    "in tenant: %s. Adding account unlock time out: %s, account lock timeout " +
-                                    "increment factor: %s raised to the power of failed login attempt cycles: %s",
-                                    userName, userStoreManager, tenantDomain, unlockTimePropertyValue,
+                                            "in tenant: %s. Adding account unlock time out: %s, account lock timeout " +
+                                            "increment factor: %s raised to the power of failed login attempt cycles: %s",
+                                    AccountUtil.maskIfRequired(userName), userStoreManager, tenantDomain,
+                                    unlockTimePropertyValue,
                                     unlockTimeRatio, currentFailedLoginLockouts);
```

Comment:
Let's take these two params to the above line.

Reply:
Resolved with https://github.com/wso2-extensions/identity-event-handler-account-lock/pull/137/commits/ae05624935d7e33a65070d5f20b5fd851e4db585